src/app/component/layout/components/search-popup/search-item/search-item.component.scss

```diff
@@ -156,8 +156,10 @@
 }
 
 @media only screen and (max-width: 576px) {
-  .title-list {
+  #list-gallery-content {
```

Comment:
Why did you use id?

Reply:
Because in previous version the wrapper has just id. Should I rewrite everything with class "list-gallery-content"?

Reply:
Fixed!